infrastructure/helm-chart/charts/api-admin/templates/deployment.yaml

```diff
@@ -22,21 +22,30 @@ spec:
     spec:
       containers:
       - name: app
-        image: ghcr.io/airyhq/api/admin:latest
+        image: "{{ .Values.image }}:{{ .Values.global.appImageTag }}"
         imagePullPolicy: Always
         env:
         - name: KAFKA_BROKERS
-          value: airy-cp-kafka-headless:9092
```

Comment:
since we're changing the names of this (which is great), I think it makes sense to drop the `airy-` from everywhere. It would be cleaner and I don't think the prefix is a requirement for anything (correct me if I'm wrong though)